### ISSUE TITLE
Add your real-world examples using floWeaver to the gallery

### ISSUE THREAD

**Issue by ricklupton** [ricklupton/floweaver#33]
We want a gallery of real-world examples of uses of floWeaver! If you have used it, please tell us about it.

The best way of contributing your example is to add it directly following the instructions below. You can also leave a comment below to tell us about your example and we can add it for you.

# Adding an example to the gallery

Examples are in the...

**Comment by arturomoncadatorres** [comment]
Hi! The the link to the CONTRIBUTING.md file is broken. I think the correct one is [this one](https://github.com/ricklupton/floweaver/blob/master/CONTRIBUTING.md).

**Comment by ricklupton** [comment]
Exactly, thanks! Did you find the process of adding your example was ok?

**Comment by arturomoncadatorres** [comment]
Yes! I had no problem at all, instructions were very clear 😉
